components/dashboard/src/contexts/FeatureFlagContext.tsx

```diff
@@ -31,15 +33,16 @@ const FeatureFlagContextProvider: React.FC = ({ children }) => {
     const team = getCurrentTeam(location, teams);
     const [showWorkspaceClassesUI, setShowWorkspaceClassesUI] = useState<boolean>(false);
     const [showPersistentVolumeClaimUI, setShowPersistentVolumeClaimUI] = useState<boolean>(false);
-
-    const featureFlags: FeatureFlagConfig = {
-        workspace_classes: { defaultValue: true, setter: setShowWorkspaceClassesUI },
-        persistent_volume_claim: { defaultValue: true, setter: setShowPersistentVolumeClaimUI },
-    };
+    const [showUsageView, setShowUsageView] = useState<boolean>(false);
 
     useEffect(() => {
         if (!user) return;
         (async () => {
+            const featureFlags: FeatureFlagConfig = {
+                workspace_classes: { defaultValue: true, setter: setShowWorkspaceClassesUI },
+                persistent_volume_claim: { defaultValue: true, setter: setShowPersistentVolumeClaimUI },
+                usage_view: { defaultValue: true, setter: setShowUsageView },
+            };
```

Comment:
What's the reason for making these very explicit (rather than reflecting over the set of flags)? I feel this has the potential to confuse others who are trying to use feature flags - having to define them here.

Reply:
Ah never mind, I missed that you just moved the definition (and the problem already exists)